samples/QuickstartWeatherServer/Tools/WeatherTools.cs

```diff
@@ -46,11 +46,19 @@ public static async Task<string> GetForecast(
         var jsonElement = jsonDocument.RootElement;
```

Comment:
> # Fix WeatherTools sample to use /forecast
> Calls to #GetForecast fail for a basic query. Periods is not present so using /Forecast attribute instead
> ...
> https://api.weather.gov/gridpoints/{office}/{gridX},{gridY}/forecast
> For example: [api.weather.gov/gridpoints/TOP/31,80/forecast](https://api.weather.gov/gridpoints/TOP/31,80/forecast)

But the sample is still calling `await client.ReadJsonDocumentAsync($"/points/{latitude},{longitude}")` one line above this. Searching for "/forecast" brings up nothing. I see this PR added some new properties like start and end time for each period. Is that all this PR is doing now? Adding a few properties?

Reply:
My bad. Initially proposed this one but seems that merge conflict went incorrectly @halter73 

```
var jsonElement = await client.GetFromJsonAsync<JsonElement>($"/points/{latitude},{longitude}");
        var forecastUrl = jsonElement.GetProperty("properties").GetProperty("forecast").GetString();
        jsonElement = await client.GetFromJsonAsync<JsonElement>(forecastUrl);
```

Makes sense?

Reply:
It seems like a reasonable improvement, but it looks like the title should probably be updated then. Should we rename this from "Fix WeatherTools sample to use /forecast" to "Return more detailed forecasts from WeatherTools"?

Reply:
Done. Updated title and code @halter73